src/views/payment-sheet-views/google-pay-view.js

```diff
@@ -100,9 +101,7 @@ GooglePayView.prototype.updateConfiguration = function (key, value) {
 };
 
 GooglePayView.isEnabled = function (options) {
-  var gatewayConfiguration = options.client.getConfiguration().gatewayConfiguration;
-
-  if (!(gatewayConfiguration.androidPay && Boolean(options.merchantConfiguration.googlePay))) {
```

Comment:
It's the name for google pay in the client configuration, which can't be changed because of backward compatibility issues in the client sdks. I believe the graphql API has the correct name for it.